packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/loggingservice/LoggingResource.java

```diff
@@ -0,0 +1,36 @@
+package software.uncharted.terarium.hmiserver.resources.loggingservice;
+
+import io.quarkus.security.Authenticated;
+import io.quarkus.security.identity.SecurityIdentity;
+import software.uncharted.terarium.hmiserver.models.loggingservice.LoggingService;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+import javax.inject.Inject;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+
+@Path("/api/")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+@Authenticated
+@Tag(name = "Model REST Endpoints")
+public class LoggingResource {
+
+	@Inject
+	SecurityIdentity securityIdentity;
+
+	@Inject
+	LoggingService loggingService;
+
+	@POST
+	@Path("/logs")
+	@Produces(MediaType.APPLICATION_JSON)
+	@Consumes(MediaType.APPLICATION_JSON)
+	public Response letsDoThis(LoggingService logData) {
```

Comment:
Also, may need a more description function name ;)

Reply:
lol, oops, yeah forgot.  glad I didn't use my usual temp naming of using curse words.